MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -210,7 +223,9 @@ class HomeViewController: UIViewController, FloatingPanelControllerDelegate, Tra
             let navBar = navController.navigationBar
 
             navBar.barTintColor = Theme.shared.colors.navigationBarBackground
-            navBar.isTranslucent = false
+            navBar.setBackgroundImage(UIImage(color: Theme.shared.colors.navigationBarBackground!), for: .default)
```

Comment:
We should not use ! in code. I think a good approach is:  color: Theme.shared.colors.navigationBarBackground ?? .black

Reply:
Any assets from the theme struct can be safely force unwrapped because I created a test that'll check all assets referenced will always be included in the bundle. 
I prefer it this way because if I fall back to the color black, the error could go unnoticed until after the app is deployed or until we implement dark mode and the text doesn't appear. The test will prevent any of those mistakes from ever getting merged and the code is simpler as well knowing you can force unwrap.
https://github.com/tari-project/wallet-ios/blob/958404e3e0f8b215d2f6a5ba5bcdf80d6e673e48/MobileWalletTests/MobileWalletTests.swift#L55